pyproject.toml

```diff
@@ -32,10 +32,21 @@ dependencies = [
 [tool.hatch.version]
 path = "src/kagglehub/__init__.py"
 
+[project.optional-dependencies]
+dev = [
```

Comment:
As discussed yesterday, is this block still needed? Probably best to standardize on always using hatch.

Reply:
sg. Hatch works natively, I will remove this.